### ISSUE TITLE
[Bug]: Disabling rotation in gestureSettings doesn't work on iOS

### ISSUE THREAD

**Issue by Elter71** [rnmapbox/maps#3676]
### Mapbox Implementation

Mapbox

### Mapbox Version

10.19.0

### React Native Version

0.76.0

### Platform

iOS

### `@rnmapbox/maps` version

10.1.33

### Standalone component to reproduce

```javascript
import React, { useEffect } from 'react';
import Mapbox from '@rnmapbox/maps';

const ShowMap = () => {
  useEffect(() => {
    Mapbox.locationManager.start();

    return (): void => {
      Mapbox.locationManager.stop();
    };
  }, []);

  return (
    <>
      <Mapbox.MapView
        styleURL="mapbox://styles/mapbox/satellite-v9"
        style={{
          flex: 1,
        }}
        testID={'show-map'}
        gestureSettings={{
          rotateEnabled: false,
          simultaneousRotateAndPinchZoomEnabled: false,
          rotateDecelerationEnabled: false,
        }}
      >
        <Mapbox.Camera followZoomLevel={12} followUserLocation />

        <Mapbox.UserLocation />
      </Mapbox.MapView>
    </>
  );
};

export default ShowMap;
```

### Observed behavior and steps to reproduce

https://github.com/user-attachments/assets/536db7de-3b8a-4f5d-96f1-ff37c1019511

### Expected behavior

When `rotateEnabled` in `gestureSettings` is set to `false`, the user can't rotate the map.

**Comment by RyanTG** [comment]
I'm experiencing this, too. Fix your lint so this can stay open! 

Also, `rotateEnabled={false}` (outside of gestureSettings) doesn't work on iOS.

**Comment by whoiscarlo** [comment]
I'm also experiencing this issue. Is there another way to stop the use from rotating the map?

I also tried the following on `MapView`:
`rotateEnabled={false}`
`pitchEnabled={false}`

But they also didn't work.

**Comment by RyanTG** [comment]
This should be fixed in v10.1.35 released today on npm.
